docs/reference.md

```diff
@@ -0,0 +1,73 @@
+# Reference
+
+---
+
+This part of the project documentation focuses on
+an **information-oriented** approach. Use it as a
+reference for the technical implementation of the
+`radcad` project code.
+
+
+## Table of Contents
+
+ - [Development](#development-subsection) 
+ - [cadCAD Compatibility](#cadcad-compatibility)
+ - [Directory Structure](#directory-structure)
+ - [Model Architecture](#model-architecture)
+ - [Known issues](#known-issues)
+
+
+---
+
+## Development
+
+Jump to [Development](development.md) section for information about:
```

Comment:
These summary sections referencing other pages are unnecessary I think and will become difficult to maintain.